FloripaPublicTransportationTests/RestApiTests.swift

```diff
@@ -0,0 +1,343 @@
+//
+//  RestApiTests.swift
+//  FloripaPublicTransportation
+//
+//  Created by Marcelo Gobetti on 2/23/16.
+//  Copyright © 2016 Marcelo Gobetti. All rights reserved.
+//
+
+import XCTest
+@testable import FloripaPublicTransportation
+import OHHTTPStubs
+
+class RestApiTests: XCTestCase, ExpectationProtocol {
```

Comment:
This did not happen to me (using latest Xcode) and the only way I could fail tests was by disconnecting my testing computer from the internet (unfortunately I found that only after creating the PR). However, the tests that fail are those related to the view controllers, the RestApiTests should run 100% fine. Interesting!